content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+      "Condition": {
+        "StringEquals": {
+          "spire-oidc-example.org:aud": "mys3",
+          "spire-oidc-example.org:sub": "spiffe://spire-oidc-example.org/workload"
+        }
+      }
+    }
+  ]
+}
+```
+
+5. Click **Update Trust Policy**.
+
+## Get a JWT SVID from the Agent and Access S3
+
+Retrieve a JWT SVID from the SPIRE Agent and use the JWT token in the JWT SVID to access S3.
+
+1. Run the following command on the instance where you've installed the SPIRE Agent to retrieve a JWT SVID:
+
+```console
+/opt/spire-agent/bin/spire-agent api fetch jwt -audience mys3 -socketPath /opt/spire-agent/sockets/agent.sock
+```
+
+This will output something similar to:
+
+```console
+token(spiffe://spire-oidc.example.org/oidc-test-workload):
+    0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789
+    JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0
+    123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789J
+    WT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN01
+    23456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JW
+    T-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN012
+    3456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT
+    -TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN012
+bundle(spiffe://spire-oidc.example.org):
+    {
+    "keys": [
+        {
+            "use": "jwt-svid",
+            "kty": "RSA",
+            "kid": "ABCDEFGHIJKLMNOPQRSTUVWXYZ012345",
+            "n": "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789
+            ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEF
+            GHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGHIJKL
+            MNOPQRSTUVWXYZ0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGHIJKLMNOPQR
+            STUVWXYZ0123456789ABCDEFGHIJKLMNOPQR",
+            "e": "AQAB"
+        }
+    ]
+}
+```
+
+2. Save the JWT portion of the previous command's output to an environmental variable, using the syntax appropriate for the shell you're using:
+```console
+export JWT=0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN012
+3456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOK
+EN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JW
+T-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456
+789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN01
+23456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TO
+KEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789JWT-TOKEN0123456789J
+WT-TOKEN0123456789JWT-TOKEN012
```

Comment:
If there's a bash command that can be used to extract the JWT portion easily, this might be worth mentioning.